src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Kindly note that all components should be class-based. Kindly refactor the app component.